```
   Compiling embedding v0.0.1 (file:///Users/servo/buildbot/slave/mac-dev-unit/build/ports/cef)
error[E0599]: no associated item named `InitializeCompositing` found for type `compositing::windowing::WindowEvent` in the current scope
   --> browser_host.rs:490:47
    |
490 |             this.downcast().send_window_event(WindowEvent::InitializeCompositing);
    |                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `embedding`.
```

--- a/components/compositing/compositor.rs
+++ b/components/compositing/compositor.rs
@@ -742,10 +742,6 @@ impl<Window: WindowMethods> IOCompositor<Window> {
                 self.composite();
             }
 
-            WindowEvent::InitializeCompositing => {
-                self.initialize_compositing();
-            }
-
             WindowEvent::Resize(size) => {
                 self.on_resize_window_event(size);
             }
@@ -1564,9 +1560,6 @@ impl<Window: WindowMethods> IOCompositor<Window> {
         } else if opts::get().is_running_problem_test {
             println!("composition_request is already {:?}", self.composition_request);
         }
-    }
-
-    fn initialize_compositing(&mut self) {
     }
 
     fn get_root_pipeline_id(&self) -> Option<PipelineId> {

--- a/components/compositing/windowing.rs
+++ b/components/compositing/windowing.rs
@@ -44,9 +44,6 @@ pub enum WindowEvent {
     /// Sent when part of the window is marked dirty and needs to be redrawn. Before sending this
     /// message, the window must make the same GL context as in `PrepareRenderingEvent` current.
     Refresh,
-    /// Sent to initialize the GL context. The windowing system must have a valid, current GL
-    /// context when this message is sent.
-    InitializeCompositing,
     /// Sent when the window is resized.
     Resize(TypedSize2D<u32, DevicePixel>),
     /// Touchpad Pressure
@@ -85,7 +82,6 @@ impl Debug for WindowEvent {
         match *self {
             WindowEvent::Idle => write!(f, "Idle"),
             WindowEvent::Refresh => write!(f, "Refresh"),
-            WindowEvent::InitializeCompositing => write!(f, "InitializeCompositing"),
             WindowEvent::Resize(..) => write!(f, "Resize"),
             WindowEvent::TouchpadPressure(..) => write!(f, "TouchpadPressure"),
             WindowEvent::KeyEvent(..) => write!(f, "Key"),

--- a/ports/cef/browser_host.rs
+++ b/ports/cef/browser_host.rs
@@ -486,8 +486,7 @@ full_cef_class_impl! {
             this.downcast().send_window_event(WindowEvent::PinchZoom((new_zoom_level / old_zoom_level) as f32))
         }}
 
-        fn initialize_compositing(&this,) -> () {{
-            this.downcast().send_window_event(WindowEvent::InitializeCompositing);
+        fn initialize_compositing(&_this,) -> () {{
         }}
 
         fn composite(&this,) -> () {{

--- a/ports/servo/main.rs
+++ b/ports/servo/main.rs
@@ -165,8 +165,6 @@ fn main() {
 
     register_glutin_resize_handler(&window, &mut browser);
 
-    browser.browser.handle_events(vec![WindowEvent::InitializeCompositing]);
-
     // Feed events from the window to the browser until the browser
     // says to stop.
     loop {

